Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:alpine AS builder
+WORKDIR /go/src/github.com/testcontainers/helloworld
+COPY . ./
+RUN go get .
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-extldflags "-static"' -o /helloworld .
+
+FROM scratch
+COPY static /static
+COPY --from=builder /helloworld /helloworld
+CMD ["/helloworld"]
```

Comment:
Why not `ENTRYPOINT`? Just an honest question 🙂

Reply:
Ah fair point. I guess it's a bit more appropriate, since we won't allow overriding the command. Will change.